### PR TITLE
virtc: Update README.md

### PR DESCRIPTION
**Pull request by sboeuf**
This PR cleans README.md according to the latest changes made to virtcontainers library.

### CODE REVIEW

File: hack/virtc/README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 
-`virtc` is a simple command-line tool that serves to demonstrate typical usage of the `virt` API.
-This is example software; unlike other projects like runc, runv, or rkt, `virt` is not a full container runtime.
+`virtc` is a simple command-line tool that serves to demonstrate typical usage of the `virtcontainers` API.
+This is example software; unlike other projects like runc, runv, or rkt, `virtcontainers` is not a full container runtime.
 
 ## Virtc example
 
-Here we explain how to use the pod API from __virtc__ command line.
+Here we explain how to use the pod and container API from __virtc__ command line.
```
Comment (dlespiau) [inline hack/virtc/README.md]: :%s/__virtc__/`virtc`/g

File: hack/virtc/README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 # virtc
 
-`virtc` is a simple command-line tool that serves to demonstrate typical usage of the `virt` API.
-This is example software; unlike other projects like runc, runv, or rkt, `virt` is not a full container runtime.
+`virtc` is a simple command-line tool that serves to demonstrate typical usage of the `virtcontainers` API.
+This is example software; unlike other projects like runc, runv, or rkt, `virtcontainers` is not a full container runtime.
```
Comment (jodh-intel) [inline hack/virtc/README.md]: I don't think "virtcontainers" needs to be rendered in a fixed-width font as it isn't a command. Also, in the context of these sentences you are referring to *projects*, not *commands*, again lending weight to not rendering in a fixed-width font.

**Comment by jodh-intel** [comment]
This looks fine, but could you make it a single commit?

**Comment by sboeuf** [comment]
@jodh-intel yes of course I will do that later because I still have to modify some things. I want to add the usage of ./virtc container ...

File: hack/virtc/README.md (hunk 2 above)
Comment (sameo) [inline hack/virtc/README.md]: Makes sense to me.

**Comment by jodh-intel** [comment]
Hi @sboeuf - any update on this? It would be good to be able to close this one off.

**Comment by sboeuf** [comment]
@jodh-intel Yeah I know but I have higher priorities for now. I will do it as soon as I can.

**Comment by sboeuf** [comment]
@sameo @jodh-intel @dlespiau I have updated it. Please take a look !

**Comment by dlespiau** [comment]
lgtm

**Comment by dlespiau** [comment]
Needs rebasing. This need of rebasing every PR is a bit of a prolem: we can't merge more than one commit at a time without human intervention and we still create a merge commit anyway.

**Comment by sboeuf** [comment]
I agree this is weird that we have to rebase even when the PRs are not related (not modifying the same files).
Is that an expected behavior for Github ? Or did we miss something in the config of the repo ?

**Comment by dlespiau** [comment]
It's a setting, I assume someone clicked that option knowingly. I just un-ticked it (we don't have that one elsewhere in our repos either).

**Pull request merged**